helm/nginx-ingress-controller-app/templates/controller-deployment.yaml

```diff
@@ -47,6 +47,23 @@ spec:
       serviceAccountName: {{ .Values.controller.name }}
       priorityClassName: system-cluster-critical
       terminationGracePeriodSeconds: {{ .Values.controller.terminationGracePeriodSeconds }}
+      initContainers:
+      # Overall performance improvements
+      # See https://github.com/kubernetes/ingress-nginx/issues/1939
+      - name: {{ .Values.initContainer.name }}
+        command:
+        - sh
+        - -c
+        - |
+          set -o errexit ; set -o xtrace ; set -o nounset
+
+          sudo sysctl -w net.core.somaxconn=32768
```

Comment:
Out of curiosity, why set it this way an not use `securityContext` of the pod?

Reply:
It's not considered a safe sysctl, so the only way to set it in the `securityContext` is to enable a kubelet flag which allows unsafe sysctls. We settled on this method as it required the least changes (kubelet flag changes = new GS release) whilst not decreasing security any more than necessary.

Reply:
Yes, correct. We could follow up with removing nginx specific settings from host, now that they are back in nginx (but do that second part in separate release, when there's really need to use newer host, e.g. flatcar based releases) https://github.com/giantswarm/k8scloudconfig/blob/master/v_6_0_0/files/conf/hardening.conf#L16-L19